woodwork/serializers/csv_serializer.py

```diff
@@ -42,7 +43,10 @@ def _get_filename(self):
         self.location = basename
         if self.data_subdirectory:
             self.location = os.path.join(self.data_subdirectory, basename)
-        return os.path.join(self.write_path, self.location)
+        location = os.path.join(self.write_path, self.location)
+        if os.path.exists(location) or glob.glob(location):
+            raise FileExistsError(location)
```

Comment:
Not totally sure what the error message for this looks like, but we should make sure it's understandable beyond letting ursers know of the offending location. Would we be able to add some explanation around why that file might already exist and what they can do to get around this error?

Reply:
Sure, what are your thoughts on:

- WoodworkFileExistsError: Typing info already exists at {location}. Please remove or use a different filename.
- WoodworkFileExistsError: Data subdirectory already exists at {location}. Please remove or use a different folder.
- WoodworkFileExistsError: Data file(s) already exists at {location}. Please remove or use a different filename.